src/iterative_sorting/iterative_sorting.py

```diff
@@ -4,23 +4,22 @@ def selection_sort( arr ):
     for i in range(0, len(arr) - 1):
```

Comment:
quick note, if you know the range will start at 0, you can exclude this, and the stop index is exclusive